x-pack/test/stack_functional_integration/configs/config.stack_functional_integration_base_ie.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resolve } from 'path';
+// import { resolve, normalize } from 'path';
+import buildState from './build_state';
+import { ToolingLog } from '@kbn/dev-utils';
+import chalk from 'chalk';
+
+const reportName = 'Stack Functional Integration Tests';
+const testsFolder = '../test/functional/apps';
+const stateFilePath = '../../../../../integration-test/qa/envvars.sh';
+const prepend = testFile => require.resolve(`${testsFolder}/${testFile}`);
+const log = new ToolingLog({
+  level: 'info',
+  writeTo: process.stdout,
+});
+
+export default async ({ readConfigFile }) => {
+  const defaultConfigs = await readConfigFile(require.resolve('../../functional/config'));
+  const { servers } = defaultConfigs.getAll();
+  const { tests, ...provisionedConfigs } = buildState(resolve(__dirname, stateFilePath));
+
+  return {
+    ...defaultConfigs.getAll(),
+    junit: {
+      reportName: `${reportName} - ${provisionedConfigs.VM}`,
+    },
+    browser: {
+      type: 'ie',
+    },
+    servers,
+    apps: {
```

Comment:
We can't run the api/status call for the w2012_ie from the host.  This change works-around that issue.
```
     servers,
+    security: { disableTestUser: true },
     apps: {
 ```

Reply:
I fix this in a recent commit